t/02mysql-parser.t

```diff
@@ -502,6 +502,19 @@ BEGIN {
     is( join(',', $t2c2->reference_fields), 'id', 'To field "id"' );
 }
 
+# Tests for CREATE VIEW statements that contain a column list
+# after the view name
+{
+    my $tr = SQL::Translator->new();
+    my $data = parse($tr,
+        q[
+            CREATE
+              VIEW view_foo (id, name) AS
+                SELECT id, name FROM thing;
+        ]
+    ) or die $tr->error;
```

Comment:
I've added better tests - it now checks VIEW statements that include 1) a field list, 2) aliases in the SELECT statement, and 3) both a field list and aliases. (When both are specified, I believe the field list should override the column aliases.)